server/pkg/item/item.go

```diff
@@ -128,3 +128,11 @@ type ItemModelSchema struct {
 	Model  *model.Model
 	Schema *schema.Schema
 }
+
+type ItemModelSchemaItemChange struct {
```

Comment:
this is not used in item package. Move it to intergrationapi package.

Reply:
@rot1024 , it is the exactly same as `ItemModelSchema` usage, which exists in the item package.

Also, this struct is used in `internal/usecase/interactor/item.go`, are you sure this should be moved?

Reply:
I see, then it should rather be defined in the use case layer. If the use case is implemented to return that struct, it should be defined in the use case layer.

Reply:
@rot1024 , after much thinking, I found out that I don't need to create a separate struct, I used existing `ItemModelSchema` with additional optional properties as needed.